pipeline_attend_and_excite.py

```diff
@@ -1,262 +1,584 @@
-from typing import List, Optional, Union, Tuple
-

```

Comment:
Could you move this pipeline into the following folder: 
https://github.com/huggingface/diffusers/tree/main/src/diffusers/pipelines/stable_diffusion

Reply:
Ah sorry just realizing that this is not the diffusers code base? :sweat_smile: 

@evinpinar could you maybe open a PR here: https://github.com/huggingface/diffusers ? :-) 

Also cc @yiyixuxu

Reply:
working on it ! :)